src/OrbitQt/orbitmainwindow.h

```diff
@@ -132,10 +132,13 @@ class OrbitMainWindow final : public QMainWindow, public orbit_gl::MainWindowInt
   orbit_base::CanceledOr<void> DisplayStopDownloadDialog(
       const orbit_client_data::ModuleData* module) override;
 
-  void SetCallTreeInspection(std::unique_ptr<CallTreeView> top_down_view,
-                             std::unique_ptr<CallTreeView> bottom_up_view) override;
+  void SetCallstackInspection(std::unique_ptr<CallTreeView> top_down_view,
+                              std::unique_ptr<CallTreeView> bottom_up_view,
+                              orbit_data_views::DataView* callstack_data_view,
+                              std::unique_ptr<class SamplingReport> report) override;
 
-  void ClearCallTreeInspection() override;
+  void ClearCallstackInspection() override;
+  void LeaveCallstackInspectionTriggered();
```

Comment:
This here is a slot instead of a signal, so I would use the prefix `On`. Or get rid of this function completely and use a lambda in the `connect`, like this:
```
  connect(ui->inspectionReport, &OrbitSamplingReport::leaveCallstackInspectionTriggered, this,
          [](){
            app_->ClearInspection();
          });
```

Reply:
Done. Thanks!